contrib/pkg/installmanager/installmanager.go

```diff
@@ -61,6 +73,13 @@ func NewInstallManagerCommand() *cobra.Command {
 				log.WithError(err).Error("invalid command options")
 				return
 			}
+
+			var err error
+			opts.KubeClient, opts.DynamicClient, err = getClients()
+			if err != nil {
+				opts.log.WithError(err).Fatal("error creating kube clients")
+			}
+
```

Comment:
Is there a reason you get the client here now instead of in the Run method?

Reply:
Felt a little like this was the constructor, but I see it's for the command not the install manager. Will move it to run.